rq/worker.py

```diff
@@ -167,6 +168,10 @@ def __init__(self, queues, name=None, default_result_ttl=DEFAULT_RESULT_TTL,
             connection = get_current_connection()
         self.connection = connection
 
+        self.redis_server_version = StrictVersion(
```

Comment:
Let's keep this to `None` and use the `get_redis_server_version()` pattern like in `queue.py` and `job.py`. This way when `Worker()` is instantiated it doesn't implicitly connect to Redis.